habitat/core/dataset.py

```diff
@@ -67,7 +69,7 @@ def scene_ids(self) -> List[str]:
         Returns:
             unique scene ids present in the dataset
         """
-        return list({episode.scene_id for episode in self.episodes})
+        return sorted(list({episode.scene_id for episode in self.episodes}))
```

Comment:
Just for completeness I think we should add a `set` constructor also here to ensure that the `scene_id` are unique.

Reply:
The curly brackets is a set construction.